### PR TITLE
Changed the alert rule of TiKV_thread_apply_worker_cpu_seconds

### PR DESCRIPTION
**Pull request by hfxsd**
### What is changed, added or deleted? (Required)

Changed the alert rule of TiKV_thread_apply_worker_cpu_seconds to the following:

`max(rate(tikv_thread_cpu_seconds_total{name=~"apply_.*"}[1m])) by (instance) > 0.9`

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:https://github.com/pingcap/docs-cn/pull/7747
- Other reference link(s):https://github.com/tikv/tikv/issues/11517

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by BusyJay — APPROVED** [review]

File: alert-rules.md [hunk 1]
```diff
@@ -596,7 +596,7 @@ This section gives the alert rules for the TiKV component.
 
 * Alert rule:
 
-    `sum(rate(tikv_thread_cpu_seconds_total{name="apply_worker"}[1m])) by (instance) > 1.8`
+    `max(rate(tikv_thread_cpu_seconds_total{name=~"apply_.*"}[1m])) by (instance) > 0.9`
```
Comment (shichun-0415) [inline alert-rules.md:599]: 告警描述中文有优化，看看英文要不要跟中文对齐.

Comment (hfxsd) [reply alert-rules.md:599]: 好的。已对齐。

File: alert-rules.md [hunk 2]
```diff
@@ -596,11 +596,11 @@ This section gives the alert rules for the TiKV component.
...
 
 * Description:
 
-    The pressure on the apply Raft log thread is too high. It is often caused by a burst of writes.
+    The pressure on the apply Raft log thread is too high, and has approached or exceeded the limit of the `apply` thread. It is often caused by a burst of writes.
```
Comment (shichun-0415) [inline alert-rules.md:603]:
```suggestion
    The apply Raft log thread suffers from high pressure and has approached or exceeded its limit. This is often caused by a burst of writes.
```
The subject of 'has approached or exceeded the limit of the `apply` thread' is 'The pressure', which is inappropriate.

**Review by shichun-0415 — APPROVED** [review]
LGTM

**Pull request merged**